binfmt/libelf/gnu-elf.ld

```diff
@@ -41,6 +41,16 @@ SECTIONS
       _etext = . ;
     }
 
+  .ARM.extab :
+    {
+      *(.ARM.extab*)
+    }
+
+  .ARM.exidx :
+    {
+      *(.ARM.exidx*)
+    }
+
```

Comment:
Seems incorrect to add it here because libelf is generic, image you compiling it for RISC-V and it trying to find these ARM sessions, should be added at ARCH specific or board linker file

Reply:
Thanks for pointing out. Added new ld file in the board specific folder.